src/biogeochem/DUSTMod.F90

```diff
@@ -293,7 +466,31 @@ subroutine DustEmission (bounds, &
          
          mbl_bsn_fct         => dust_inst%mbl_bsn_fct_col            , & ! Input:  [real(r8) (:)   ]  basin factor
          flx_mss_vrt_dst     => dust_inst%flx_mss_vrt_dst_patch      , & ! Output: [real(r8) (:,:) ]  surface dust emission (kg/m**2/s)               
-         flx_mss_vrt_dst_tot => dust_inst%flx_mss_vrt_dst_tot_patch    & ! Output: [real(r8) (:)   ]  total dust flux back to atmosphere (pft)
+         flx_mss_vrt_dst_tot => dust_inst%flx_mss_vrt_dst_tot_patch  , & ! Output: [real(r8) (:)   ]  total dust flux back to atmosphere (pft)
```

Comment:
@dmleung I noticed that Leung no longer has a dependence on u10 anymore, unlike Zender. I just wanted to make sure that was correct.

If so I'll remove the setting of u10 here as it's unreferenced.

Reply:
Yes! We don't use u10 in Leung. It's only used in Zender when calculating the so-called Owen's effect. Feel free to take it out if you want to.